### ISSUE TITLE
Local Dev: Status stays unscanned, can't open attachment

### ISSUE THREAD

**Issue by DanSchlachter** [cap-js/attachments#84]
With SAPUI5 version 1.125.1 the status of an attachment stays in status unscanned in local development with `scan : false`.
When clicking on the uploaded attachment after save , it gives an error
``` xml
<code>403</code>
<message>Unable to download the attachment as scan status is not clean.</message>
```

**Comment by muskansethi1** [comment]
Hi @DanSchlachter,

With PR #83, we would be not having this check when Malware Scanning is disabled, hence the file will be accessible.
This would be available in the next release.

Best Regards,
Muskan Sethi

**Comment by muskansethi1** [comment]
Hi,

We have released the fix for this issue in version 1.1.5

Best Regards,
Muskan
